packages/react-jss/src/createHoc.js

```diff
@@ -57,41 +74,49 @@ let managersCounter = 0
 /**
  * Wrap a Component into a JSS Container Component.
  *
+ * IProps: Props of the InnerComponent.
```

Comment:
InnerProps would be probably a better name, no?

Reply:
already using InnerProps though, that's why I had to name it IProps 🤔

Reply:
lets find a better convention, maybe InnerPropsType?